index.bs

```diff
@@ -2751,7 +2751,7 @@ Council Decision Report</h5>
 	The [=Council=] <em class=rfc2119>may</em> also issue a <dfn export>Supplemental Confidential Council Report</dfn>
 	with a more restricted <a href="#confidentiality-levels">level of confidentiality</a> than its main report
 	when it believes that additional commentary on confidential aspects of the case
-	would be informative.
+	would be non-normative.
```

Comment:
In this instance, "informative" definitely means "carrying (interesting) information", not "non-normative", so the replacement is not appropriate.